mailbox/cassandra/src/main/java/org/apache/james/mailbox/cassandra/modules/CassandraApplicableFlagsModule.java

```diff
@@ -35,7 +35,7 @@
         .comment("Holds flags being used on specific mailboxes. As system flags are implicit, this table " +
             "stores user flags.")
         .options(options -> options
-            .compactionOptions(SchemaBuilder.leveledStrategy())
+            .compactionOptions(SchemaBuilder.sizedTieredStategy())
```

Comment:
what was your reasoning to move from LCS to STS ?
Can you explain it in the commit description.

https://docs.datastax.com/en/dse/5.1/dse-dev/datastax_enterprise/config/configChooseCompactStrategy.html